src/bundler/bundle_v2.zig

```diff
@@ -1164,20 +1157,18 @@ pub const BundleV2 = struct {
         // conditions from creating two
         _ = JSC.WorkPool.get();
 
-        if (!BundleThread.created) {
-            BundleThread.created = true;
-
+        if (BundleThread.instance) |existing| {
+            existing.queue.push(completion);
+            existing.waker.?.wake();
```

Comment:
This is technically a race condition, I haven't been able to hit it, but if you stress test the bundler it is theoretically possible to hit this before the waker thread has actually started. I'm not sure how to properly wait for this, or maybe this can be an if()|waker|?